application/modules/admin/views/scripts/shipment/index.phtml

```diff
@@ -11,6 +11,7 @@
         </a>
     </div>
 </h4>
+<input type="hidden" id="scheme" name="scheme" value="<?php echo $this->schemes[0]['scheme_id']; ?>"  >
```

Comment:
The [S1EPT-20 epic](https://ihsinformatics.atlassian.net/browse/S1EPT-20) is about making it quicker to create a panel by refactoring the concept of schemes out of the system. This system was original forked off of [victormwenda/ept](https://github.com/victormwenda/ept) which is where the concept of schemes comes from.

For our purposes, we only deal with _Dried Tube Specimen - Tuberculosis_ panels.

For the tickets in this epic, instead of adding hidden fields to pass through unnecessary validation rules, please can we remove the validation rules and default the shipment's scheme_id to *tb* on the back end?